backend/src/main/java/com/company/project/service/TimetableService.java

```diff
@@ -26,7 +26,7 @@ public List<TimetableDto> getTimetable() {
         return timeSlotListToTimetableList(timetableEntities);
     }
 
-    private static List<TimetableDto> timeSlotListToTimetableList(List<Timeslot> timetableEntities) {
+    public static List<TimetableDto> timeSlotListToTimetableList(List<Timeslot> timetableEntities) {
```

Comment:
kradzież mojej prywatnej funkcji 🫨

Reply:
Nie powiedziałbym, że to kradzież towarzyszu. Zastanawiałem się nad dodaniem do dependencji mappera i stworzeniem klas do konwersji entity do dto.

Reply:
Haha, też o tym myślałem i ogółem tam by się przydało zrobić albo właśnie mapper albo funkcje które robią konwersje, ale przyznam, że dosyć trudno się to ręcznie pisało i dużo czasu spędziłem, żeby te szalone streamy napisać, więc może mapper byłby lepszy?

Reply:
Dobra, jutro ogarnę mapper+wyjątki+@data